internal/cmd/common/common.go

```diff
@@ -22,6 +22,25 @@ const (
 	FulcioPrefix = "fulcio:"
 )
 
+// PathList is a custom type to represent a list of paths
+type PublicKeys []string
+
+// String implements part of the pflag.Value interface.
+func (p *PublicKeys) String() string {
+	return strings.Join(*p, ",")
```

Comment:
> How is this method used for other types? Such as the default ones? Should it in fact provide the value? Is this meant to be used for user facing messages? If so, I'd make the separator `, ` perhaps, with a space?
This is not actual used anywhere, it is just needed to satisfy the pflag.Value interface for cobra. I think it could potentaily be used in more complex cobra commands that I do not know about, and the default types have this implmented for them in the cobra library. 
> But IMO we need to figure this out, because a path on disk may not be very useful, unless this is primarily meant for local logging or so.
I think we should open up an issue on this, to finaly decide how the passing of keys into gittuf should work, and in what format.